<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfigBooks</class>
 <widget class="QDialog" name="DialogConfigBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>لوحة تحكم الكتب</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>حدد قسما من القائمة المنسدلة</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="comboBoxSection"/>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonSectionAdd">
             <property name="toolTip">
              <string>اضافة قسم جديد</string>
             </property>
             <property name="statusTip">
              <string>اضافة قسم جديد</string>
             </property>
             <property name="whatsThis">
              <string>اضافة قسم جديد</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/image/list-add.png</normaloff>:/images/image/list-add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonSectionRname">
             <property name="toolTip">
              <string>اعادة تسمية القسم</string>
             </property>
             <property name="whatsThis">
              <string>اعادة تسمية القسم</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/image/edit-rename.png</normaloff>:/images/image/edit-rename.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonSectionRemove">
             <property name="toolTip">
              <string>حذف القسم المحدد</string>
             </property>
             <property name="whatsThis">
              <string>حذف القسم المحدد</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/image/edit-delete.png</normaloff>:/images/image/edit-delete.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>قائمة المجموغات</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QTreeWidget" name="treeWidgetCurGroup">
           <column>
            <property name="text">
             <string notr="true">المجموعة</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QToolButton" name="toolButtonGroupAdd">
             <property name="toolTip">
              <string>اضافة مجموعة جديدة</string>
             </property>
             <property name="whatsThis">
              <string>اضافة مجموعة جديدة</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/image/list-add.png</normaloff>:/images/image/list-add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonGrouprename">
             <property name="toolTip">
              <string>اعادة تسمية الحجموعة المحددة</string>
             </property>
             <property name="whatsThis">
              <string>اعادة تسمية الحجموعة المحددة</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/image/edit-rename.png</normaloff>:/images/image/edit-rename.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonGroupRemove">
             <property name="toolTip">
              <string>حذف المجموعة المحددة</string>
             </property>
             <property name="whatsThis">
              <string>حذف المجموعة المحددة</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/image/edit-delete.png</normaloff>:/images/image/edit-delete.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonGroupTosection">
             <property name="toolTip">
              <string>نقل المجموعة الى قسم محدد</string>
             </property>
             <property name="whatsThis">
              <string>نقل المجموعة الى قسم محدد</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="arrowType">
              <enum>Qt::RightArrow</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>قائمة الكتب</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QTreeWidget" name="treeWidgetCurBooks">
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <column>
            <property name="text">
             <string notr="true">الكتاب</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QToolButton" name="toolButtonRemoveBooks">
             <property name="toolTip">
              <string>حذف الكتب المحددة</string>
             </property>
             <property name="whatsThis">
              <string>حذف الكتب المحددة</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/image/edit-delete.png</normaloff>:/images/image/edit-delete.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonBookToGroup">
             <property name="toolTip">
              <string>نقل الكتب المحددة الى المجموعة المحددة</string>
             </property>
             <property name="whatsThis">
              <string>نقل الكتب المحددة الى المجموعة المحددة</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="arrowType">
              <enum>Qt::RightArrow</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>حدد كتابا لتتمكن من نحرير بياناته</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidgetBooks">
         <column>
          <property name="text">
           <string notr="true">شجرة الكتب</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widgetBk" native="true">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>اذا كان لديك ديل يحتوي على كتب لا توجد لها قائمة او فقدت القائمة يمكنك انشاء قائمة للكتب بالنقر على الزر بحيث سيقوم البرنامج بالبحث والتصنيف تلقائيا ثم يمكنك تعديلها بعد ذلك اما اذا كانت النتائج غير مرضية فيمكنك استعادة الاعدادات السابقة من الزر تحديث</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolButton" name="toolButtonFindBKOld">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonReloadAll">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>مسار الكتب</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonBkPath">
           <property name="toolTip">
            <string>تغيير مسار الكتب</string>
           </property>
           <property name="whatsThis">
            <string>تغيير مسار الكتب</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>قائمة الكتب</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditGroup">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonOPenGroup">
           <property name="toolTip">
            <string>تغيير قائمة الكتب المستخدمة حاليا</string>
           </property>
           <property name="whatsThis">
            <string>تغيير قائمة الكتب المستخدمة حاليا</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonGroupUpdat">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>استعادة قائمة الكتب السابقة</string>
           </property>
           <property name="whatsThis">
            <string>استعادة قائمة الكتب السابقة</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>دليل الكتاب</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditBkPath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonRenBkPath">
           <property name="toolTip">
            <string>تغيير اسم دليل الكتاب</string>
           </property>
           <property name="whatsThis">
            <string>تغيير اسم دليل الكتاب</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/image/edit-rename.png</normaloff>:/images/image/edit-rename.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>صورة الكتاب</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonImg">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>96</height>
            </size>
           </property>
           <property name="toolTip">
            <string>اختيار صورة للكتاب</string>
           </property>
           <property name="whatsThis">
            <string>انقر هنا لاختيار صورة مخصصة للكتاب</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/image/booki.png</normaloff>:/images/image/booki.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>اسم الكتاب</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditBooknam"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>اسم المؤلف</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditAautor"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>بطاقة الكتاب</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textBrowserBetaka">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonUpdatBk">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>تحديث بيانات الكتب</string>
         </property>
         <property name="whatsThis">
          <string>تحديث بيانات الكتب</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConfigBooks</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConfigBooks</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
